### ISSUE TITLE
Video Pauses after a few seconds but audio keeps playing 0.15.2 still an issue

### ISSUE THREAD

**Issue by DwayneGoddem** [jellyfin/jellyfin-androidtv#2248]
### Describe the bug

When using the default video players on some videos the picture will freeze and the sounds will keep going.
This is on a Firestick 4K latest client and Jellyfin Server.

Then using a 3rd party player (Kodi or VLC) I have the same error. Until I DISABLE Hardware Acceleration. The firestick seams to have an issue with this.  I confirmed the same issue using Kodi and VLC using SMB shares has the same issues until the hardware acceleration is disabled.

Can we have an option to disable the acceleration with libvlc etc as this might fix the playback issues on the Firesticks.

Yes I have also tested with with remote clients also on Firesticks.

This is still an issue with 0.15.2.

We need a way to disable hardware acceleration on the firestick.

### Media info of the file

Ghostbusters 720P from YTS will do this every time just after the intro

### Application version

0.15.2

### Device information

Amazon Firestick 4k

### Android version

Firestick Android 

### Jellyfin server version

10.8.7+

**Comment by AnharMiah** [comment]
I have the exact same issue! FireStick on some videos will freeze, but audio continues :(

**Comment by DwayneGoddem** [comment]
This is still an issue with 0.15.3 client.

I have not found a way to even force disable hardware acceleration as a system wide or if there is any way to access the Libvlc template file to add the commands to disable hardware acceleration

libvlc_media_add_option(m, ":avcodec-hw=none");

**Comment by HijackHornet** [comment]
I have this issue too on some mkv videos on the firestick 4kmax. It seems to only happens on some files as I managed to watch 200 episodes before I ever had this issue.

**Comment by deepdivered** [comment]
similar issue but disabling hardware acceleration did not fix mit but changing to 1mbs does

**Comment by diegohvanni** [comment]
Same issue here. However, affects only a handful of *.mkv files. 
None of the client options seem to make a difference, but for some videos, decreasing the bitrate to under 3Mbps works.

This issue is driving me nuts. I Already tried everything I could imagine in terms of cofig on both sides (server and client) nothing works.

**Comment by DwayneGoddem** [comment]
This is still an ongoing issue.

With the latest client version

**Comment by ninthbit** [comment]
Running server 10.9.4 with FireStick 4k Max using Jellyfin 0.16.11.  I've got the same issue.  However, I'm testing with my copy of The Mummy.  My 4k Max freezes at 8:15 consistently.  My non-Max 4k Firesticks don't freeze.  All of them are using Exoplayer.

**Comment by DwayneGoddem** [comment]
> Running server 10.9.4 with FireStick 4k Max using Jellyfin 0.16.11. I've got the same issue. However, I'm testing with my copy of The Mummy. My 4k Max freezes at 8:15 consistently. My non-Max 4k Firesticks don't freeze. All of them are using Exoplayer.

That is right it's only a 4k MAX thing. They never fixed the hardware acceleration issue.

**Comment by teslazap** [comment]
I have a fresh install of Jellyfin 10.9.11 using Docker/Portainer and I have the same issue browsing with VLC from windows: movie starts, video hangs after a few seconds and audio keeps going. Seeking doesn't work either.
I don't have hardware acceleration enabled, wifi bandwith is more than adequate for this kind of streaming.

**Comment by JonathanHolvey** [comment]
I appear to be having a very similar problem direct streaming live TV from a TVHeadend server to Jellyfin for Android TV v0.18.0. The video stream will play fine for a little while, then the picture freezes and the audio continues. I can't find any relevant information in the Android adb logcat logs, nor in the Jellyfin server's system logs.

What can I do to help diagnose this issue better?

**Comment by DwayneGoddem** [comment]
> I appear to be having a very similar problem direct streaming live TV from a TVHeadend server to Jellyfin for Android TV v0.18.0. The video stream will play fine for a little while, then the picture freezes and the audio continues. I can't find any relevant information in the Android adb logcat logs, nor in the Jellyfin server's system logs.
> 
> What can I do to help diagnose this issue better?

In our case it is an issue with the Firestick and and playing video that have a bad frame. using the hardware decoding it will not skip over the frame and the video just crashes.

when using software decoding it played just fine. So using VLC with hardware decoding disabled works around the problem.

So this will not help with your issue sorry.

**Comment by JonathanHolvey** [comment]
@DwayneGoddem  I'm using a Raspberry Pi 4, which allows you to disable hardware decoding in the Android TV settings. Since disabling it, the issue has gone. Even if it's a different cause, I'm happy that the solution is the same. Thanks.
